packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -244,15 +226,14 @@ export class DetailsRowBase extends BaseComponent<IDetailsRowBaseProps, IDetails
         role="row"
         aria-label={ariaLabel}
         ariaDescribedBy={ariaDescribedBy}
-        className={css(classNames.root)}
+        className={classNames.root}
```

Comment:
Rm'ed redundant `css()`

Reply:
Hmm, I question the need for the one on **new** line 274 too. And I'm curious if the rest should eventually be removed with their logic moved to `DetailsRow.styles.ts`?

Reply:
Yea I think so, this is in a half state. Let me evaluate this offline and raise a PR separately for the remainder.